opentelemetry-api/src/opentelemetry/loader.py

```diff
@@ -61,7 +61,10 @@ def my_factory_for_t(api_type: typing.Type[T]) -> typing.Optional[T]:
 means that the Python interpreter was invoked with the ``-E`` or ``-I`` flag).
 """
 
-from typing import Callable, Optional, Type, TypeVar
+from typing import Callable
+from typing import Optional
+from typing import Type
+from typing import TypeVar
```

Comment:
This is uglier than combined imports, but to use isort we need the same style: either single lines or combined everywhere. Without this change isort was failing for other files.

Reply:
It actually looks better to me.